Backend/Remora.Discord.API.Abstractions/API/Objects/Permissions/DiscordPermission.cs

```diff
@@ -241,5 +241,15 @@ public enum DiscordPermission
     /// Allows for timing out users to prevent them from sending or reacting to messages in chat and threads, and from
     /// speaking in voice and stage channels.
     /// </summary>
-    ModerateMembers = 40
+    ModerateMembers = 40,
+
+    /// <summary>
+    /// Allows for viewing role subscription insights.
+    /// </summary>
+    ViewCreatorMonetizationAnalytics = 41,
+
+    /// <summary>
+    /// Allows for using soundboard in a voice channel.
+    /// </summary>
+    UseSoundboard = 42
```

Comment:
This permission should be added as a forwarded member to the DiscordVoicePermission enum as well.